packages/jellyfish-api-core/src/category/governance.ts

```diff
@@ -129,13 +129,20 @@ export class Governance {
   /**
    * Returns information about proposal votes.
    *
-   * @param {string} proposalId Proposal id
-   * @param {MasternodeType | string} [masternode=MasternodeType.MINE] masternode id or reserved words 'mine' to list votes for all owned accounts or 'all' to list all votes
-   * @param {number} [cycle=0] cycle: 0 (show current), cycle: N (show cycle N), cycle: -1 (show all)
+   * @param {ListGovProposalVotesOptions} options
+   * @param {string} options.proposalId Proposal id
+   * @param {MasternodeType | string} [options.masternode=MasternodeType.MINE] masternode id or reserved words 'mine' to list votes for all owned accounts or 'all' to list all votes
+   * @param {number} [options.cycle=-1] cycle: 0 (show current), cycle: N (show cycle N), cycle: -1 (show all)
+   * @param {ListGovProposalVotesPagination} [options.pagination]
+   * @param {number} [options.pagination.start] default is 0
+   * @param {boolean} [options.pagination.including_start] default = true
```

Comment:
```suggestion
   * @param {number} [options.pagination.start = 0]
   * @param {boolean} [options.pagination.including_start = false]
```

You can specify defaults in JSDoc.

Reply:
Fixed: https://github.com/JellyfishSDK/jellyfish/pull/1908/commits/09c483e9cd769cbf47fbce486d20bca448c6bf91